src/main/java/com/hillel/items_exchange/dto/UserDto.java

```diff
@@ -38,6 +36,6 @@
     private String avatarImage;
     @NotNull(message = "{invalid.not-null}")
     @PastOrPresent(message = "{invalid.past-or-present.date}")
-    @JsonFormat(pattern = "yyyy-MM-dd")
-    private LocalDate lastOnlineTime;
+    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
+    private LocalDateTime lastOnlineTime;
 }
```

Comment:
@lubchik-14, could you add list of the phones and children to UserDTO? @Wolshebnik is started working on the personal area and he needs that data into one request.